replica/table.cc

```diff
@@ -650,7 +668,7 @@ class tablet_storage_group_manager final : public storage_group_manager {
             }
             // FIXME: don't allocate compaction groups for tablets that aren't present in this shard.
```

Comment:
remove?

Reply:
or do you still allocate

Reply:
I'll leave it for @Deexie, in her PR. we're still allocating all tablet ids, as before.